src/internal/operations/restore.go

```diff
@@ -175,8 +180,10 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 
 	dcs, err := op.kopia.RestoreMultipleItems(ctx, bup.SnapshotID, paths, opStats.bytesRead)
 	if err != nil {
-		opStats.readErr = errors.Wrap(err, "retrieving service data")
-		return nil, opStats.readErr
+		op.Errors.Fail(errors.Wrap(err, "retrieving service data"))
+		opStats.readErr = op.Errors.Err()
```

Comment:
Same question as L168 --> Do we need a guard check in case op.Errors is nil

Reply:
no